README.md

```diff
@@ -7,6 +7,16 @@ Contao Swiper
 
 Contao 4+ bundle to integrate [Swiper.js](http://idangero.us/swiper/).
 
+This extension replaces the core slider with a Swiper implementation.
+
 ## Usage
 
-__jQuery__ needs to be enabled in your page layout. This extension replaces the core slider with a Swiper implementation.
+__jQuery__ does **not** have to be enabled in your page layout.
```

Comment:
I think you should leave the comment

> This extension replaces the core slider with a Swiper implementation.

since that is still true.

Reply:
It is still there: [in Line 10](https://github.com/fritzmg/contao-swiper/pull/3/files/4e5f0178641f7f667b0bc59d2e092100ca403829#diff-04c6e90faac2675aa89e2176d2eec7d8R10)

Reply:
Ha, sorry, did not see that 😁